### DISCUSSION TITLE
Proposing to add a video tutorial prepared by me

### DISCUSSION THREAD

**Discussion by FahimFBA** [OpenBangla/OpenBangla-Keyboard#276] [Ideas]
Hello, tonight I have prepared a YouTube video where I provided all the step-by-step guidelines starting from the installation to adding iBUS to the Autostart. The video is still on `UNLISTED`, as I am asking you to check the video whether you would add the video to your project as well for the beginners. 

The link to the video: https://youtu.be/C8vnR1AY6sA

**Addition**: I will also prepare a complete guideline about this on the international platform later according to my plan.

**Last update till now**: I am making the video public.

**Comment by mominul** [comment]
আপনার ভিডিওটি বেশ ভাল লেগেছে আমার! আসলে এরকম ভিডিও দরকার অনেকেরই, তাই আপনার উদ্যোগের জন্য বিশেষভাবে ধন্যবাদ :heart: 
 
যেহেতু ভিডিওটি আর্চ স্পেসিফিক, তাই আমার মনে হয় আপনি README এবং [উইকিয়ের Arch installation instructions](https://github.com/OpenBangla/OpenBangla-Keyboard/wiki/Installing-OpenBangla-Keyboard#arch-linux-and-its-derivatives) এ যুক্ত করতে পারেন ভিডিও লিংকটি।

ধন্যবাদ এবং শুভ কামনা! :blush:

**Comment by mominul** [comment]
আর সামনের ৩.০.০ রিলিজে installation পদ্ধতি বেশ বদলে যাবে (নতুন fcitx backend যুক্ত হতে চলেছে) তাই ভবিষ্যতে নতুন করে কিছু ভিডিও বানাতে হতে পারে আপনাকে যদি viewers দের আপডেটেড রাখতে চান।

এখন আমি নিশ্চিতভাবে বলতে পারছি না যে কবে ৩.০.০ রিলিজ করতে পারব, তাই আমি আপনাতে অপেক্ষাও করতে বলতে পারছিনা। তবে নতুন ভার্সন আসার আগ পর্যন্ত অনেকেই ভিডিও গুলো থেকে উপকার পাবেন বলে আমি আশা করি :blush:

**Comment by FahimFBA** [comment]
আমি ভাবছিলাম ডেব, আরপিএম এর ক্ষেত্রেও আমি ভিডিও তৈরী করব। আর যেহেতু ৩.০.০ রিলিজে ইন্সটলেশন পদ্ধতি পরিবর্তন হবে, সেহেতু নতুন রিলিজের পরপরেই আমি আপডেটেড ভিডিও তৈরী করব ইনশাআল্লাহ। আপনাদের এপ্রিশিয়েশন দেখে আমি আসলেই মুগ্ধ। 😀

**Comment by mominul** [comment]
@FahimFBA আপনাকেও স্বাগতম! :blush:
